src/lcals/FIRST_MIN-Cuda.cpp

```diff
@@ -168,22 +168,23 @@ void FIRST_MIN::runCudaVariantRAJANewReduce(VariantID vid)
 
   if ( vid == RAJA_CUDA ) {
 
-    using VL_TYPE = RAJA::expt::ValLoc<Real_type>;
-
     startTimer();
     for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-       VL_TYPE tloc(m_xmin_init, m_initloc);
+      RAJA::expt::ValLoc<Real_type, RAJA::Index_type> tminloc(m_xmin_init,
+                                                              m_initloc);
 
        RAJA::forall<exec_policy>( res,
          RAJA::RangeSegment(ibegin, iend),
-         RAJA::expt::Reduce<RAJA::operators::minimum>(&tloc),
-         [=] __device__ (Index_type i, VL_TYPE& loc) {
-           loc.min(x[i], i);
+         RAJA::expt::Reduce<RAJA::operators::minimum>(&tminloc),
+         [=] __device__ (Index_type i,
+           RAJA::expt::ValLocOp<Real_type, Index_type,
```

Comment:
The `ValLocOp` `Index_type` should match the above `ValLoc`'s `RAJA::Index_type`. I can't quite tell if they match, but we should check. It's probably working due to type erasure in the internals.

Reply:
Good catch -- thanks!. That's a cut-and-paste error.  The types are the same and the types in RAJAPerf are aliases to the RAJA types. I pushed a change plus some others for code consistency.